beacon_node/beacon_chain/Cargo.toml

```diff
@@ -61,6 +61,7 @@ execution_layer = { path = "../execution_layer" }
 sensitive_url = { path = "../../common/sensitive_url" }
 superstruct = "0.5.0"
 hex = "0.4.2"
+exit-future = "0.2.0"
```

Comment:
Do we need this guy?

Reply:
We use him in `beacon_chain::test_utils`.

Reply:
If this is just testing? Can it be a dev-dep then?

Reply:
It's just for testing, but it's not hidden behind a `#[test]` compilation flag so it must be a major dependency. We can't hide it behind a `#[test]` flag since it's used across multiple crates and Cargo doesn't play nicely with that.